### PR TITLE
issue #11126 - Add sample code to TextFormField.

### PR DESCRIPTION
**Pull request by branflake2267**
https://github.com/flutter/flutter/issues/11126

### CODE REVIEW

File: packages/flutter/lib/src/material/text_form_field.dart [hunk 1]
```diff
@@ -27,6 +27,23 @@ import 'text_field.dart';
...
+///     icon: const Icon(Icons.person),
+///     hintText: 'What do people call you?',
+///     labelText: 'Name *',
+///   ),
+///   onSaved: (String value) {},
+///   validator: (String value) {
+///     if (value.contains("@")) return "Do not use the @ char.";
+///     return null;
+///   },
+/// );
```
Comment (cbracken) [inline packages/flutter/lib/src/material/text_form_field.dart]: Drop the trailing `;`

File: packages/flutter/lib/src/material/text_form_field.dart [hunk 2]
```diff
@@ -27,6 +27,23 @@ import 'text_field.dart';
...
+/// ```dart
+/// new TextFormField(
+///   decoration: const InputDecoration(
+///     icon: const Icon(Icons.person),
+///     hintText: 'What do people call you?',
+///     labelText: 'Name *',
+///   ),
+///   onSaved: (String value) {},
+///   validator: (String value) {
+///     if (value.contains("@")) return "Do not use the @ char.";
```
Comment (cbracken) [inline packages/flutter/lib/src/material/text_form_field.dart]: Use single-quotes here for style consistency with the above/other examples.

Comment (cbracken) [inline packages/flutter/lib/src/material/text_form_field.dart]: Break the `return` onto the next line, or alternatively you could use a ternary:
```
return value.contains('@') ? 'Input may not contain @.' : null;
```

**Review by cbracken — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a few minor nits.

**Comment by Hixie** [comment]
@branflake2267 Are you planning on applying the suggested fixes or should we take the PR over? (Either way is fine, just checking so we don't step on your toes.)

**Comment by Hixie** [comment]
@branflake2267 I'm going to close this PR for now, but please feel free to resubmit it if you get a chance to work on it again. If you don't we'll get to it eventually.

**Comment by branflake2267** [comment]
Oops, I had lost this in my noise. Sounds good. :)

**Comment by goderbauer** [comment]
@jslavitz Can we close this now that your PR has been submitted?

**Pull request closed without merging**